modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/NavigationBarTag.java

```diff
@@ -98,11 +99,21 @@ protected int processStartTag() throws Exception {
 		if (_navigationItems != null) {
 			JspWriter jspWriter = pageContext.getOut();
 
-			jspWriter.write("<div class=\"container-fluid ");
-			jspWriter.write("container-fluid-max-xl\"><div ");
+			jspWriter.write("<div class=\"container-fluid");
+
+			if (!FeatureFlagManagerUtil.isEnabled("LPS-184404")) {
+				jspWriter.write(" container-fluid-max-xl");
+			}
+
+			jspWriter.write("\"><div ");
 			jspWriter.write("class=\"collapse navbar-collapse\"><div ");
-			jspWriter.write("class=\"container-fluid ");
-			jspWriter.write("container-fluid-max-xl\"><ul ");
+			jspWriter.write("class=\"container-fluid");
+
+			if (!FeatureFlagManagerUtil.isEnabled("LPS-184404")) {
+				jspWriter.write(" container-fluid-max-xl");
+			}
+
+			jspWriter.write("\"><ul ");
```

Comment:
Looks like at least the `<ul ` could be on the next line with the rest of it's element. Maybe the whole thing if the SF doesn't complain.